src/Microsoft.TemplateEngine.Cli/New3Command.cs

```diff
@@ -180,6 +178,11 @@ private IFilteredTemplateInfo UseSecondaryCriteriaToDisambiguateTemplateMatches(
 
         public static int Run(string commandName, ITemplateEngineHost host, Action<IEngineEnvironmentSettings, IInstaller> onFirstRun, string[] args)
         {
+            if (args.Any(x => string.Equals(x, "--debug:attach", StringComparison.Ordinal)))
+            {
+                Console.ReadLine();
```

Comment:
Thinking about this - since this is the console project (and getting this to run from F5 in VS/VS Code is much more difficult than running it with setup/explicit launch), I think we can hold off on this one for now.